imgui/src/utils.rs

```diff
@@ -173,4 +173,15 @@ impl<'ui> Ui<'ui> {
     pub fn style_color(&self, style_color: StyleColor) -> [f32; 4] {
         self.ctx.style()[style_color]
     }
+    /// Returns a shared reference to the current [`Style`]. This function is tagged as `unsafe`
```

Comment:
We should note that users who would like to avoid unsafe can use `clone_style()`.